### PR TITLE
Ensure JSON is wrapped in quotations

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: templates/docker_importer.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
 {
     <% unless [nil, :undefined, :undef, ''].include?(scope['pulp::proxy_url']) -%>
-    "proxy_host": <%= scope['pulp::proxy_url'] %>,
+    "proxy_host": "<%= scope['pulp::proxy_url'] %>",
     <% else %>
     "proxy_host": null,
     <% end %>
     <% unless [nil, :undefined, :undef, ''].include?(scope['pulp::proxy_port']) -%>
-    "proxy_port": <%= scope['pulp::proxy_port'] %>,
+    "proxy_port": "<%= scope['pulp::proxy_port'] %>",
```
Comment (jlsherrill) [inline templates/docker_importer.json]: Port needs to NOT have quotes

**Comment by stbenjam** [comment]
Not exactly related but if the templates had a `.erb` extension the diffs would look better

**Comment by ehelms** [comment]
@stbenjam you are the winner!

**Comment by ehelms** [comment]
Updated to fix tests, keep port a number and change to erb

**Comment by jlsherrill** [comment]
ACK

**Comment by jlsherrill** [comment]
merging to make sure this makes nightly, thanks @ehelms !

**Pull request merged**